test/test_env.py

```diff
@@ -1946,6 +1946,32 @@ def test_rollout_policy(self, batch_size, rollout_steps, count):
             else:
                 assert (td["lazy"][..., i]["action"] == 0).all()
 
+    @pytest.mark.parametrize("batch_size", [(), (1, 2), (2,)])
+    @pytest.mark.parametrize("env_type", ["serial", "parallel"])
+    def test_vec_env(self, batch_size, env_type, rollout_steps=4, n_workers=2):
```

Comment:
each parallel test takes a long time to launch. Are all the combinations absolutely necessary? Can we have just one Parallel?